website/cue/reference/services/splunk_client.cue

```diff
@@ -0,0 +1,8 @@
+package metadata
+
+services: splunk_client: {
+	name:     "Splunk client"
+	thing:    "a \(name) client"
```

Comment:
Would this render as `a Splunk client client`?
```suggestion
	thing:    "a \(name)"
```

Reply:
A good catch :)